lib/tbot/tbot.go

```diff
@@ -224,7 +225,15 @@ func (b *Bot) Run(ctx context.Context) error {
 		return trace.Wrap(err)
 	}
 
-	if err := b.initialize(ctx); err != nil {
+	unlock, err := b.initialize(ctx)
+	defer func() {
+		if unlock != nil {
+			if err := unlock(); err != nil {
+				b.log.WithError(err).Warn("Failed to release lock. Future starts of tbot may fail.")
```

Comment:
Is this true? When this tbot process exits, the lock should be released by the kernel (according to flock(2)). I wonder if this log message may instil unnecessary worry in users.

Reply:
This is certainly true for `flock` - however, we are looking into supporting other destinations such as Kubernetes secrets etc, and for these, the automagic behaviour of `flock` will probably not necessarily apply.